vsutil/_util.py

```diff
@@ -0,0 +1,15 @@
+"""
+Private functions not meant to be exported by the main package.
+"""
+__all__ = []
```

Comment:
Just leave it out.

Reply:
Oh woops, yea was going to fill that with something but forgot I left it there.